cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -31,7 +32,9 @@ public Capacity(KafkaSpec spec) {
         BrokerCapacity bc = spec.getCruiseControl().getBrokerCapacity();
 
         this.diskMiB = bc != null && bc.getDisk() != null ? getSizeInMiB(bc.getDisk()) : generateDiskCapacity(spec.getKafka().getStorage());
-        this.cpuUtilization = bc != null && bc.getCpuUtilization() != null ? bc.getCpuUtilization() : DEFAULT_BROKER_CPU_UTILIZATION_CAPACITY;
+        this.cpuUtilization = bc != null && bc.getCpuUtilization() != null ? bc.getCpuUtilization() :
+                (spec.getKafka().getResources() == null || spec.getKafka().getResources().getLimits() == null ||
+                        spec.getKafka().getResources().getLimits().get("cpu") == null) ? DEFAULT_BROKER_CPU_UTILIZATION_CAPACITY : Quantities.parseCpuAsMilliCpus(spec.getKafka().getResources().getLimits().get("cpu").toString()) / 10;
```

Comment:
Why do you use limit and not request? Also, why do you divide it by 10?

Reply:
Cruise Control will crash if we provide values outside the interval [0,100], see my comment below

Reply:
@sknot-rh please ignore my previous comment, Cruise Control will not crash if we provide values outside the interval [0,100] but the utilization numbers don't seem to make sense when we do